### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by Kampftroll**
- Updated german translation
- _originally used JSON in commit , now using JSON FLAT again_

- _Im using Traduora translation management platform, the exported JSON FLAT is restructured from the original used by this repository, if that becomes an issue, please let me know_

### CODE REVIEW

**Review by Dmitry221060 — CHANGES_REQUESTED** [review]
Hello, thank you for your contribution!
Can you please keep the original format of the json file? Keeping the structure consistent with other lang files will make it much easier to navigate and compare, and the original structure has a logical ordering to make it easier to work with (Tabs and elements are in the same order you would see them in the mod menu, dialogs are in the order you would encounter them, etc)

File: assets/rpvoicechat/lang/de.json [hunk 1]
```diff
@@ -27,10 +27,23 @@
...
+    "Gui.AudioWizardDialog.SecondParagraph": "Konfigurieren Sie nun Ihre Aufnahmelautstärke. Drücken Sie die Taste unten und beginnen Sie wie gewohnt zu sprechen. Sprechen Sie so lange, bis der Fortschrittsbalken gefüllt ist. Andere Spieler können Sie im Moment nicht hören, also machen Sie sich keine Sorgen um sie.",
+    "Gui.AudioWizardDialog.Done": "Das war's! Die Lautstärke Ihrer Aufnahme wurde automatisch kalibriert, viel Spaß beim Spielen.",
+    "Gui.Button.NotNow": "Nicht jetzt",
+    "Gui.Button.Sure": "Okay",
+    "Gui.FirstLaunchDialog.TitleBar": "Willkommen beim RPVoiceChat!",
+    "Gui.FirstLaunchDialog.FirstParagraph": "Es sieht so aus, als ob Sie den RPVoiceChat mod zum ersten Mal benutzen. Möchten Sie Ihr Eingabegerät automatisch konfigurieren lassen? Es wird weniger als eine Minute dauern.",
+    "Gui.FirstLaunchDialog.SecondParagraph": "Dieses Fenster wird nur einmal angezeigt, aber Sie können den Konfigurationsassistenten jederzeit über das Mod-Menü (\"{0}\"-Taste) aufrufen.",
```
Comment (Dmitry221060) [inline assets/rpvoicechat/lang/de.json:46]: Does this say "from the mod menu" or "using the mod menu"? Given that this is first time player uses the mod they may think that pressing P would be enough to open audio wizard and may get a bit lost as to how to proceed

File: assets/rpvoicechat/lang/de.json [hunk 2]
```diff
@@ -27,10 +27,23 @@
...
+    "Command.EncodeAudio.Success.Enabled": "Audiocodierung ist aktiviert",
+    "Command.EncodeAudio.Success.Disabled": "Audiocodierung ist deaktiviert",
     "Hotkey.ModMenu": "RPVoice: Mod Menü",
     "Hotkey.VoiceLevel": "RPVoice: Sprachdistanz ändern",
     "Hotkey.PTT": "RPVoice: Push to talk",
     "Hotkey.Mute": "RPVoice: Stumm schalten",
+    "Gui.AudioWizardDialog.TitleBar": "Audio-Assistent",
+    "Gui.AudioWizardDialog.FirstParagraph": "Willkommen beim Audiokonfigurationsassistenten, der Ihnen helfen wird, Ihr Eingabegerät zu konfigurieren. Sie sollten sich jetzt hören können. Wenn das nicht der Fall ist, stellen Sie sicher, dass dies Ihr Gerät ist:",
+    "Gui.AudioWizardDialog.SecondParagraph": "Konfigurieren Sie nun Ihre Aufnahmelautstärke. Drücken Sie die Taste unten und beginnen Sie wie gewohnt zu sprechen. Sprechen Sie so lange, bis der Fortschrittsbalken gefüllt ist. Andere Spieler können Sie im Moment nicht hören, also machen Sie sich keine Sorgen um sie.",
```
[the remark this answers is not in the document]
Comment (Kampftroll) [reply assets/rpvoicechat/lang/de.json:40]: Yeah, no it is really formal, i can change it sure!

File: assets/rpvoicechat/lang/de.json [hunk 3]
```diff
@@ -27,10 +27,23 @@
...
+    "Gui.AudioWizardDialog.Done": "Das war's! Die Lautstärke Ihrer Aufnahme wurde automatisch kalibriert, viel Spaß beim Spielen.",
+    "Gui.Button.NotNow": "Nicht jetzt",
+    "Gui.Button.Sure": "Okay",
```
[the remark this answers is not in the document]
Comment (Kampftroll) [reply assets/rpvoicechat/lang/de.json:43]: Yes, we can use "Sicher", which is more casual, like in "Na sicher können wir das"

File: assets/rpvoicechat/lang/de.json (hunk 1 above)
Comment (Kampftroll) [reply assets/rpvoicechat/lang/de.json:46]: It says, "You can call up the configuration wizard at any time via the Mod menu (p)."

Comment (Dmitry221060) [reply assets/rpvoicechat/lang/de.json:46]: Sounds a bit ambiguous but I think it will be fine 👍

**Review by Dmitry221060 — APPROVED** [review]
Tested in game, looks great 👍

Merge with squash.

**Pull request merged**